content/manuals/build/builders/_index.md

```diff
@@ -78,27 +78,21 @@ older versions of the Docker CLI. The `docker buildx build` command, on the
 other hand, checks whether you've set a different builder as the default
 builder before it sends your build to BuildKit.
 
-To use the `docker build` command with a non-default builder, you must either:
+To use the `docker build` command with a non-default builder, you must either
+specify the builder explicitly:
 
-- Specify the builder explicitly, using the `--builder` flag or the `BUILDX_BUILDER` environment variable:
+- Using the `--builder` flag:
 
   ```console
-  $ BUILDX_BUILDER=my_builder docker build .
   $ docker build --builder my_builder .
   ```
 
-- Configure Buildx as the default client by running the following command:
+- Or the `BUILDX_BUILDER` environment variable:
 
   ```console
-  $ docker buildx install
+  $ BUILDX_BUILDER=my_builder docker build .
   ```
 
-  This updates your [Docker CLI configuration file](/reference/cli/docker/_index.md#configuration-files)
-  to ensure all of your build-related commands are routed via Buildx.
-
-  > [!TIP]
-  > To undo this change, run `docker buildx uninstall`.
-
```

Comment:
Maybe add:
```
> [!NOTE]
> An experimental command `docker buildx install` that allowed aliasing `docker buildx build` to `docker build` is deprecated since October 2025, and will be removed in a future release.
```